src/iop/colorbalancergb.c

```diff
@@ -41,8 +41,8 @@
 
 //#include <gtk/gtk.h>
 #include <stdlib.h>
-#define LUT_ELEM 360 // gamut LUT number of elements: resolution of 1°
-#define STEPS 72     // so we test 72×72×72 combinations of RGB in [0; 1] to build the gamut LUT
+#define LUT_ELEM 360     // gamut LUT number of elements: resolution of 1°
+#define STEPS 92         // so we test 72×72×72 combinations of RGB in [0; 1] to build the gamut LUT
```

Comment:
pedantic comment - wouldn't that make the test 92x92x92 instead of mentioned in comment 72x72x72?

Reply:
yeah, right.